### ISSUE TITLE
Truncated PDF417 not detected

### ISSUE THREAD

**Issue by Ted3535** [googlesamples/android-vision#335]
I'm impressed with the performance of the Mobile Vision barcode library vs that of ZXing, but MV is not able to read my truncated PDF417 barcodes, whereas ZXing does. These barcodes exist on CAC cards used by the US military. "Truncated" means they lack the stop pattern.

Here's a website that allows generating such truncated barcodes: http://www.abarcode.net/online.aspx?barcode=PDF417

For reference, I've attached truncated and non-truncated barcodes with text "Test Test Test Test".

I'm testing this on a fully updated Galaxy S6 with Google Play Services 12.2.21.

From my build.gradle: compile 'com.google.android.gms:play-services-vision:11+'

I've taken a scan of a CAC card with its truncated barcode and added the stop pattern using an image editor, and the printed result scans correctly with MV.

Any help would be appreciated. Thanks.

![barcode-non truncated](https://user-images.githubusercontent.com/37848449/37991083-0c8a0022-31bd-11e8-96c9-b812a9f63177.png)

![barcode-truncated](https://user-images.githubusercontent.com/37848449/37991084-0ca1f092-31bd-11e8-98d9-8ccb3be084ae.png)

![cac-card-190x300](https://user-images.githubusercontent.com/37848449/37991217-7a0cfe92-31bd-11e8-88e6-4b56648d9855.jpg)

**Comment by kee23** [comment]
Are there any plans to support this in the future? I would like this feature as well.

**Comment by CarterAAelect** [comment]
I agree, that would be a great addon to an already great library. :)

**Comment by red5leader** [comment]
Bumping the issue here. Is this implemented at all or will be implemented. It's a fairly common barcode in retail especially for tickets.
